lib/dlopen.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const binding = process.binding('dlopen');
+
+exports.dlopen = function dlopen(name) {
+  var lib = new Buffer(binding.sizeof_uv_lib_t);
```

Comment:
Sorry man, still not sure what you're driving at.

Are you worried about automating `dlclose()` with when `lib` get GC'd? I'm not so worried about that, I consider that the user's responsibility.